doc/configuration/privacyidea-appliance.rst

```diff
@@ -112,14 +112,54 @@ Restore
 The entry `View Backups` will list all the backups available.
 
 .. figure:: images/appliance/backup3.png
-   :scale: 50%
+   :width: 550px
 
    All available backups
 
 You can select a backup and you are asked if you want to restore the data. 
 
 .. warning:: Existing data is overwritten and will be lost.
 
+.. _database:
+
+Database: Setup Redundancy
+..........................
+
+The appliance-tool is also capable of setting up a redundant setup between
+two privacyIDEA nodes. For the setup, the privacyIDEA slave instance has to
+be setup and running with identical software than the master instance. The
```

Comment:
For the setup, you only need to install the privacyIDEA packages in the same version on the second node. It is not necessary to configure anything on the second node.

Reply:
I rewrote the paragraph.